device/globals/interactive.gd

```diff
@@ -109,17 +111,27 @@ func _process(time):
 		var angle = old_pos.angle_to_point(pos)
 		set_position(pos)
 
-		last_dir = _get_dir(angle)
+		last_dir = _get_dir_deg(angle)
 
-		if has_node("animation"):
+		if animation:
 			if last_dir != -1 && animation.get_current_animation() != animations.directions[last_dir]:
 				animation.play(animations.directions[last_dir])
 
-		#pose_scale = animations.directions[last_dir+1]
-
 	if self is esc_type.ITEM:
 		_update_terrain()
 
+func turn_to(deg):
+	if deg < 0 or deg > 360:
+		vm.report_errors("interactive", ["Invalid degree to turn to " + String(deg)])
+
+	last_dir = _get_dir_deg(deg)
+
+	if animation:
+		if !animation.get_current_animation() or animation.get_current_animation() != animations.directions[last_dir]:
+			animation.play(animations.directions[last_dir])
+
+			set_state(animations.idles[last_dir])
```

Comment:
Does this have any effect?

Reply:
Ok, I believe you; it just wasn't clear to me what purpose it would serve in this context. :slightly_smiling_face: Might be a good place to add a comment?

Reply:
Fine, I'll be force-pushing a comment.